lib/NetPacksLib.cpp

```diff
@@ -1208,6 +1208,11 @@ DLL_LINKAGE void BattleObstaclePlaced::applyGs( CGameState *gs )
 	gs->curB->obstacles.push_back(obstacle);
 }
 
+DLL_LINKAGE void BattleDrawbridgeStateChanged::applyGs(CGameState *gs)
+{
+	gs->curB->si.drawbridgeState = state;

```

Comment:
Should be check for curB. State change on the end of battle in theory may lead to crash.